feg/protos/s6a_proxy.proto

```diff
@@ -168,7 +168,7 @@ message UpdateLocationAnswer {
 
     NetworkAccessMode network_access_mode = 8;
 
-   bytes regional_subscription_zone_code = 9;
+    repeated string regional_subscription_zone_code = 9;
```

Comment:
is it just me or is the spacing weird here?

Reply:
I think it was off before